magento/magento2
# #27091 Removed array print while setup upgrade

### Description (*)
This PR helps to remove array print while setup upgrade & installation.

### Fixed Issues (if relevant)
1. fixes magento/magento2#27091

### Manual testing scenarios (*)

1. php bin/magento s:up
2. we shouldn't see printed array content for caches (enabling/disabling)
![removed-print-array](https://user-images.githubusercontent.com/18184256/78392760-64b9c900-7606-11ea-9298-f4036c0eddaf.png)
3. Check unit test by `vendor/bin/phpunit -c dev/tests/unit/phpunit.xml.dist setup/src/Magento/Setup/Test/Unit/Model/InstallerTest.php`

### Questions or comments
If any please let me know your feedback

### Contribution checklist (*)
